CPS.md

```diff
@@ -930,21 +930,21 @@ ISRG CA Private Keys are generated inside HSMs and are only transferred between
 
 ISRG CA Private Keys are stored on HSMs meeting the requirements stated in Section 6.2.1.
 
-### 6.2.8 Method of activating private key
+### 6.2.8 Activating Private Keys
 
 ISRG CA Private Keys are always stored on HSMs and activated using the mechanisms provided by the HSM manufacturer.
 
-### 6.2.9 Method of deactivating private key
+### 6.2.9 Deactivating Private Keys
 
 ISRG CA Private Keys are always stored on HSMs and deactivated using the mechanisms provided by the HSM manufacturer.
 
-### 6.2.10 Method of destroying private key
+### 6.2.10 Destroying Private Keys
 
 ISRG CA Private Keys are destroyed by Trusted Contributors using a FIPS 140-2 (or higher) validated zeroize method provided by the HSMs storing the keys. Physical destruction of the HSM is not required.
 
 See the Let's Encrypt Subscriber Agreement for information regarding Subscriber private key destruction.
 
-### 6.2.11 Cryptographic Module Rating
+### 6.2.11 Cryptographic Module Capabilities
```

Comment:
This is a tough call. This change would make us match the CP but it would cause us to deviate from RFC 3647, which we say we follow.

Reply:
Oh, I see. That also applies to most/all of the modified sections beyond what you highlighted here. Maybe a counterargument is that: RFC 3647 sets forth a "recommended outline" that is "strongly advised," but short of a MUST; none of these changes affect the substance; and it may be slightly less confusing / more consistent to exactly match the CP (and therefore the BRs).

I don't feel strongly either way on this myself, but if we want to exactly match RFC 3647 instead, then there would be a few other changes.